about.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" />
+    <title>About:Capstone-I-Project</title>
+  </head>
+  <body>
+    <nav id="Mobile" class="Mobile-menu">
+      <div class="humburger">
+        <i class="fa-solid fa-bars"></i>
+      </div>
+    </nav>
+    <nav>
+      <div class="first-bar">
+        <div>
+          <a href="#">
+            <i class="fa-brands fa-facebook-f"></i>
+          </a>
+        </div>
+        <div>
+          <a href="#">
+            <i class="fa-brands fa-twitter"></i>
+          </a>
+        </div>
+        <div>
+          <a href="#">English</a>
+        </div>
+        <div>
+          <a href="#">My Page</a>
+        </div>
+        <div>
+          <a href="#">Logout</a>
+        </div>
+      </div>
+      <div class="seconde-bar">
+        <a class="Logo" href="index.html">
+          <img src="images/logo44.png" alt="logo-website">
```

Comment:
### OPTIONAL 

- [ ] Kindly reduce logo size to look similar to the Behance design 👍🏿 

Behance Design          |  Your design    
:-------------------------:|:-------------------------:
![Screenshot_265](https://user-images.githubusercontent.com/69638013/174903159-a4b0885f-a53d-4030-90bd-dd0a3c6809f7.png)|  ![Screenshot_266](https://user-images.githubusercontent.com/69638013/174903192-4137f7cc-ff60-4a3d-aa9f-949ed4aa5e67.png)